<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-CFNR1QK5" failures="1" tests="6" name="Test" time="131.93" errors="0" timestamp="2024-05-19T13:21:34 IST">
  <testcase classname="TestCases.RegisterTest" name="VerifyRegsiterValidData" time="4.843"/>
  <testcase classname="TestCases.RegisterTest" name="VerifyRegsiterInvalidData" time="4.543"/>
  <testcase classname="TestCases.RegisterTest" name="VerifyRegsiterEmptyData" time="14.574"/>
  <testcase classname="TestCases.LoginTestCase" name="VerifyLoginValidData" time="2.348"/>
  <testcase classname="TestCases.LoginTestCase" name="VerifyLoginInValidData" time="16.819"/>
  <testcase classname="TestCases.LoginTestCase" name="VerifyLoginEmptyData" time="1.665">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found [Warning: No match for E-Mail Address and/or Password.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found [Warning: No match for E-Mail Address and/or Password.]
at resources.commonMethod.handlingAssertion(commonMethod.java:16)
at TestCases.LoginTestCase.VerifyLoginEmptyData(LoginTestCase.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginEmptyData -->
  <testcase classname="resources.BaseClass" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="resources.BaseClass" name="@BeforeSuite ExtentReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite ExtentReport -->
</testsuite> <!-- Test -->
